tardis/montecarlo/tests/test_montecarlo.py

```diff
@@ -75,61 +73,6 @@
     assert_allclose,
 )
 
-from tardis import __path__ as path
-
-
-@pytest.fixture(scope="module")
-def continuum_compare_data_fname():
-    fname = "continuum_compare_data.hdf"
-    return os.path.join(path[0], "montecarlo", "tests", "data", fname)
-
-
-@pytest.fixture(scope="module")
-def continuum_compare_data(continuum_compare_data_fname, request):
-    compare_data = pd.HDFStore(continuum_compare_data_fname, mode="r")
-
-    def fin():
-        compare_data.close()
```

Comment:
I wonder why these fixtures were added(added in this pull request: https://github.com/tardis-sn/tardis/pull/1319). @Rodot- @chvogl @andrewfullard do we intend to make use of these fixtures at some point in time?

Here is the file: https://github.com/tardis-sn/tardis/blame/master/tardis/montecarlo/tests/test_montecarlo.py#L82

Reply:
I believe these fixtures were added as part of the continuum physics additions but the tests haven't been completed since we're still waiting on NLTE effects to be finished.

Reply:
@Rodot-  should we go ahead with their removal or keep them?

Reply:
Yes, we should keep these!